index.js

```diff
@@ -711,6 +779,11 @@
     return Array$prototype$equals.call(this, other);
   }
 
+  //  Arguments$prototype$lte :: Arguments ~> Arguments -> Boolean
+  function Arguments$prototype$lte(other) {
+    return Array$prototype$lte.call(this, other);
+  }
+
   //  Error$prototype$toString :: Error ~> () -> String
   function Error$prototype$toString() {
```

Comment:
Let's remove `RegExp$prototype$lte` for now. As Irakli rightly pointed out, we can easily add `fantasy-land/lte` implementations for Error and RegExp at some point in the future.